src/ddmd/todt.d

```diff
@@ -719,7 +719,7 @@ private void membersToDt(AggregateDeclaration ad, DtBuilder dtb,
         {
             dtb.xoff(toVtblSymbol(concreteType), 0);  // __vptr
             offset = Target.ptrsize;
-            if (!cd.classKind == ClassKind.cpp)
+            if (cd.classKind != ClassKind.cpp)
```

Comment:
Should we look for more of these?

Reply:
I did. I didn't find any.  Feel free to check yourself.